packages/@react-spectrum/menu/docs/ActionMenu.mdx

```diff
@@ -0,0 +1,259 @@
+<!-- Copyright 2021 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/menu';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '@react-spectrum/menu/package.json';
+
+```jsx import
+import {ActionMenu, Item, Section} from '@react-spectrum/menu';
+import Copy from '@spectrum-icons/workflow/Copy';
+import Cut from '@spectrum-icons/workflow/Cut';
+import {Flex} from '@react-spectrum/layout';
+import Paste from '@spectrum-icons/workflow/Paste';
+```
+
+---
+category: Collections
+keywords: [action menu, dropdown]
+after_version: 3.5.0
+---
+
+# ActionMenu
+
+<p>{docs.exports.ActionMenu.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['ActionMenu']}
+  sourceData={[
+    {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/popover/'}
+  ]} />
+
+## Example
+
+```tsx example
+<ActionMenu>
+  <Item>Cut</Item>
+  <Item>Copy</Item>
+  <Item>Paste</Item>
+</ActionMenu>
+```
+
+## Content
+
+ActionMenu follows the [Collection Components](../react-stately/collections.html) API,
+accepting both static and dynamic collections. See [Menu](Menu.html#content) for further explanation.
+
+```tsx example
+function Example() {
+  let actionMenuItems = [
+    {name: 'Cut'},
+    {name: 'Copy'},
+    {name: 'Paste'},
+    {name: 'Replace'}
+  ];
+
+  return (
+    <ActionMenu items={actionMenuItems}>
+      {item => <Item key={item.name}>{item.name}</Item>}
+    </ActionMenu>
+  );
+}
+```
+
+### Trays
+On mobile, ActionMenus automatically display in a tray instead of a popover to improve usability.
+
+### Internationalization
+In order to internationalize an AcitonMenu, the content provided to all child items should be localized.
+For languages that are read right-to-left (e.g. Hebrew and Arabic), the layout of the ActionMenu is flipped.
+
+## Events
+ActionMenu supports the `onAction` callback which receives the selected item. See [Selection](#selection) for more information.
```

Comment:
No Selection section on this page, should this link to Menu's docs?

Reply:
ActionMenu doesn't support selection, `onSelectionChange()` or `selectedKeys` so it seemed like this section should be unique and not point to Menu.

Reply:
```suggestion
ActionMenu supports the `onAction` prop, which is called when the user presses a menu item. It receives the key of the item as an argument, which can be used to perform the relevant action.
```